runtimes/CMakeLists.txt

```diff
@@ -17,6 +17,31 @@ list(INSERT CMAKE_MODULE_PATH 0
   "${CMAKE_CURRENT_SOURCE_DIR}/../llvm/cmake/modules"
 )
 
+# Variable naming for instrumented libraries
+function(get_cxx_name_sufix OUT_VAR  USE_SANITIZER)
+  set(USE_SANITIZER "${USE_SANITIZER}")
+  if (USE_SANITIZER STREQUAL "")
+    set(${OUT_VAR} "" PARENT_SCOPE)
+  elseif (USE_SANITIZER STREQUAL "Address")
+    set(${OUT_VAR} "asan" PARENT_SCOPE)
+  elseif (USE_SANITIZER STREQUAL "HWAddress")
+    set(${OUT_VAR} "hwasan" PARENT_SCOPE)
+  elseif (USE_SANITIZER MATCHES "Memory(WithOrigins)?")
+    set(${OUT_VAR} "memory" PARENT_SCOPE)
```

Comment:
it looks inconsistent

if we use asan,  hwasan, then
msan, ubsan, tsan, dfsan - short names used in other parts

not sure uasan, we didn't have short name for these pair
ubasan

also we need ubhwasan or uhwasan, as they can be paired same way